### DISCUSSION TITLE
Use Aspect Ratio similarly to require('windicss/plugin/aspect-ratio')

### DISCUSSION THREAD

**Discussion by Phyrom** [unocss/unocss#1909] [General]
I am working moving from windicss to unocss. I have a case that I use aspect ratio provided by require('windicss/plugin/aspect-ratio'). 

Is there anyway we can achieve that in Unocss?

Thank you

**Comment by chu121su12** [comment]
Aspect ratio is in preset-mini

https://github.com/unocss/unocss/blob/3d805df256f7b2632ad205155e6eff4a56a3b992/test/assets/preset-mini-targets.ts#L514-L524
